xamarin/xamarin-macios
# Merge latest d16-2 into xcode11

Changes:

 a3b8cddd3 (monojenkins, 8 days ago)
    Ignore the outgenerated projects until they are moved to their own dir.
    (#6714)

    Related issue https://github.com/xamarin/xamarin-macios/issues/6162
 2e925029a (monojenkins, 13 days ago)
    [d16-2] [tests] wrench is long gone (#6698)

    * [tests] wrench is long gone

    * Use a different Url that works and ensure we get a nice message when we
    have a failure.

    * Fix new callback test.

 c66911641 (monojenkins, 2 weeks ago)
    [package] Remove creation of files/symlinks under /Developer (#6665)

 72cb587a3 (monojenkins, 5 weeks ago)
    [d16-2] [GameController] GCGamepad is deprecated in the headers, so mark it
    as such. Fixes xamarin/maccore#1742. (#6428)

    * [GameController] GCGamepad is deprecated in the headers, so mark it as
    such. Fixes xamarin/maccore#1742.
     Fixes https://github.com/xamarin/maccore/issues/1742.

    * Add attribute for tvOS too.

 4e37f75ec (monojenkins, 5 weeks ago)
    [tests] Only call MTLDevice.CreateSharedTexture macOS 10.14+. Fixes
    xamarin/maccore#1663. (#6423)

    Fixes https://github.com/xamarin/maccore/issues/1663.
 3415f7005 (monojenkins, 5 weeks ago)
    [xharness] Don't try to find simulators for ignored tests. (#6421)

    Trying to find a simulator will mark the test as a failure if the simulator
     couldn't be found, and we don't want that to happen to ignored tests.
     This should fix an issue where xharness seems to try to run the 32-bit
    simulator tests when asked to run only device tests.
 a8bcecced (Manuel de la Pena, 7 weeks ago)
    [d16-2] Bump mono to pick tz/DST fixes.. (#6454)

 97c552e56 (monojenkins, 8 weeks ago)
    [CoreGraphics] Fix lifetime of delegates passed to CGPatternCreate. (#6368)

    We may get callbacks after the managed CGPattern instance has been
    disposed/garbage collected, so make sure our delegates survives that long.
     Since the delegates don't need any instance state, just make them static.
 34b324fac (monojenkins, 8 weeks ago)
    [tests] Handle Xcodes without proper versions by ignoring them. Fixes
    xamarin/maccore#1768. (#6386)

    This can happen if an Xcode hasn't been completely/successfully installed
    or removed.
     Fixes https://github.com/xamarin/maccore/issues/1768.
